index.html

```diff
@@ -0,0 +1,70 @@
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div>
+      <main id="main">
+        <h1 id="title">
+          Dr. Normal Borlaug
+        </h1>
+        <p>The man who saved a billion lives</p>
+        <div id="img-div">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image" alt="Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.">
+         <figcaption id="img-caption">
+           Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+         </figcaption>
+        </div>
+        <div id="tribute-info">
+          <h3>Here's a time line of Dr. Borlaug's life:</h3>
+          <ul>
+            <li>
+              <strong>1914</strong>
+              - Born in Cresco, Iowa
+            </li>
+            <li>
+              <strong>1933</strong>
+               - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"
+            </li>
+            <li>
+              <strong>1935</strong>
+               - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+            </li>
+            <li>
+              <strong>1938</strong>
+              - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+            </li>
+          </ul>
+          <em>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</em><br><br>
+          <em>--Indian Prime Minister Manmohan Singh</em>
+        </div>
+        <h3>If you have time, you should read more about this incredible human being on his <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank" id="tribute-link">Wikipedia entry</a>.</h3>
+      </main>
+      </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    <p>Hello World!</p> than fatch by using git pull command than
+    <p>Hello world, we are a web developer!</p>
+    than git add index.html
+    git commit -m write a commit_messege
+    git push
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
- Code is not well organized.
- Images should have alt text for screen readers to handle them.
- Body tag has no sections for different parts of the code.
- Proper GitHub workflow has not been followed.
- HTML and CSS best practices are recommended.

Reply:
- I think it will be best to rephrase your comment here as well in order to make them more polite and friendly. 🤗 
- It will be great to state reasons WHY a change needs to be implemented 🍒 
- I'd like to also suggest that you remember to add some emojis as well 🍷 

_Why don't we re-phrase these 5 comments you have added above??_